components/CourseViewer/CourseDetail.tsx

```diff
@@ -687,21 +750,21 @@ class CourseDetailImpl extends JCComponent<Props, JCState> {
                 color: "#333333",
               }}
             >
-              {lesson.date}
+              {lesson?.date}
```

Comment:
@GeorgeBellTMH the `date` field does not exist in the CourseLesson table. Is this supposed to be something else? Should I just get rid of this `<Text/>` node? 

Same thing in all four `render<LessonType>()` functions.

Reply:
Probably can get rid of it for now...

Reply:
https://github.com/jesus-collective/mobile/pull/739